roles/tas_single_node/tasks/certificates.yml

```diff
@@ -169,6 +169,7 @@
             old_root_content != tas_single_node_fulcio.root_ca
             and (tas_single_node_fulcio.private_key != ""
               or (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_fulcio_auto_generated_private_key) | list | length) == 0)
+          changed_when: false
```

Comment:
This `changed_when` shouldn't really be possible IMO. Assuming the `when` logic works fine, this task will only trigger if a certain config change happened, but rerunning again after that shouldn't trigger it again, should it?